priority_variables_transform/ARIC-ingest/cause_of_death.yaml

```diff
@@ -7,6 +7,8 @@
         associated_visit:
           value: ARIC Death Event
           range: string
+        age_at_death:
+          populated_from: {phv00203832} * 365
```

Comment:
All of the new `age_at_death` with `populated_from:` in this file should be `expr:`